src/common/low_precision_transformations/src/strided_slice.cpp

```diff
@@ -129,12 +148,17 @@ bool StridedSliceTransformation::canBeTransformed(const TransformationContext& c
     }
 
     if (operation->get_input_partial_shape(0).rank().is_dynamic() &&
-        ((dequantization.subtract && ngraph::shape_size(dequantization.subtractConstant->get_shape()) > 1) ||
-         (dequantization.multiply && ngraph::shape_size(dequantization.multiplyConstant->get_shape()) > 1))) {
+        ((dequantization.subtract && shape_size(dequantization.subtractConstant->get_shape()) > 1) ||
+         (dequantization.multiply && shape_size(dequantization.multiplyConstant->get_shape()) > 1))) {
         return false;
     }
 
-    return true;
+    OPENVINO_SUPPRESS_DEPRECATED_START
+    return
+        ov::get_constant_from_source(operation->get_input_source_output(1)) &&
+        ov::get_constant_from_source(operation->get_input_source_output(2)) &&
+        ov::get_constant_from_source(operation->get_input_source_output(3));
```

Comment:
If dequantization constants are scalar, it's no matter whether begins/ends/strides are constant or not. I suggest checking that only in case of per channel dequantization

Reply:
good point, done